### ISSUE TITLE
include_vars - directory inconsistency

### ISSUE THREAD

**Issue by simon-sinx17** [ansible/ansible-modules-core#5205]
##### ISSUE TYPE

- Bug Report
##### COMPONENT NAME

include_vars
##### ANSIBLE VERSION

```
ansible 2.1.2.0
  config file = /home/user/project/ansible.cfg
  configured module search path = Default w/o overrides
```
##### CONFIGURATION

N/A
##### OS / ENVIRONMENT

N/A
##### SUMMARY

Inconsistent directory handling by include_vars module 
##### STEPS TO REPRODUCE

```
structure:

site.yml -> include: /playbooks/test.yml
/playbooks/test.yml -> roles: test 
/roles/test/tasks/main.yml -> include_vars: "{{ vars_file | default() }}"

```

##### EXPECTED RESULTS

consistency
##### ACTUAL RESULTS

```
include_vars: "{{ vars_file | default() }}"
fatal: [hostname]: FAILED! => {"failed": true, "msg": "the file_name '/home/user/project/roles/test/tasks' does not exist, or is not readable"}

include_vars: "{{ vars_file | default('../file.yml') }}"
fatal: [hostname]: FAILED! => {"changed": false, "failed": true, "file": "/home/user/project/file.yml", "invocation": {"module_args": {"_raw_params": "../file.yml"}, "module_name": "include_vars"}, "msg": "Source file not found."}

include_vars: "{{ vars_file | default('file.yml') }}"
fatal: [hostname]: FAILED! => {"changed": false, "failed": true, "file": "/home/user/project/playbooks/file.yml", "invocation": {"module_args": {"_raw_params": "file.yml"}, "module_name": "include_vars"}, "msg": "Source file not found."}

works as expected with absolute path
```

**Comment by ansibot** [comment]
@bennojoy ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@bennojoy, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@bennojoy, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@simon-sinx17: Thanks for taking the time to open this issue, but it was filed under the wrong github repository. Please create a new issue at https://github.com/ansible/ansible-modules-ansible/issues
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
